Gems/ROS2/Code/Source/RobotImporter/URDF/ArticulationsMaker.cpp

```diff
@@ -53,8 +54,8 @@ namespace ROS2
             if (jointAxis != nullptr)
             {
                 jointCoordinateAxis = URDF::TypeConversions::ConvertVector3(jointAxis->Xyz());
-                quaternion =
-                    jointCoordinateAxis.IsZero() ? AZ::Quaternion::CreateIdentity() : AZ::Quaternion::CreateShortestArc(o3deJointDir, jointCoordinateAxis);
+                quaternion = jointCoordinateAxis.IsZero() ? AZ::Quaternion::CreateIdentity()
+                                                          : AZ::Quaternion::CreateShortestArc(o3deJointDir, jointCoordinateAxis);
```

Comment:
Consider : AZ::Vector3::CreateAxisX() and removing `o3deJointDir`

Reply:
Fixed.